spec/requests/api/milestones_spec.rb

```diff
@@ -71,6 +78,13 @@
       json_response['title'].should == 'updated title'
     end
 
+    it "should update a project milestone by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.